### DISCUSSION TITLE
C404 Bias Adjustment DEM

### DISCUSSION THREAD

**Discussion by ghost** [hytest-org/hytest#530] [Data Storage and Access]
Hi HyTest,

Is there a DEM used for the bias adjustment process? A DEM is referenced in the metadata on sciencebase, but I can't find one on Hovenweep Caldera or in the GLOBUS endpoint linked from the data release. 

Thank you!
-Theo

**Comment by sfoks** [comment]
@theobarnhart-USGS, do you want the DEM which was used to determine elevation for the adjustment? Thinking of this sentence below from the [C404BA data release](https://www.sciencebase.gov/catalog/item/64f77acad34ed30c20544c18) metadata:

>"_...Then, the bias in temperature values within each CONUS404 1 kilometer cell were adjusted based on the grid cell elevation. For grid cells less than 2000 meters in mean elevation, daily temperature was adjusted using a bias adjustment factor associated with the day of the year in which it is simulated to occur._ "

**Comment by ghost** [comment]
Thank you, @sfoks! Yes, that DEM would be great. I'm working on making SnowModel forcing files from this dataset and need the elevation for each grid cell.

**Comment by aubreyd** [comment]
Hi @theobarnhart-USGS!  I am confirming with the team (particularly @YongxinFred) so don't take this as truth yet, but my guess is that they used the HGT_M field in the WRF-Hydro/Noah-MP geogrid file. You can find that in the [WRF-Hydro data release](https://doi.org/10.5066/P1KZGLU2). Go to the Globus endpoint then /NIWAA_200910_202110_ScienceBase_Data/static_input_files/WRFHydro_geo_em.d01.CONUS_1km_NIWAAv1.0.nc. If you need the spatial reference info you can find that in the same directory under WRFHydro_GEOGRID_LDASOUT_Spatial_Metadata_CONUS_1km_NIWAAv1.0.nc

There is also the 4-km DEM that was part of the original CONUS404 WRF simulations - I can try to find a link to that file if useful.

**Reply by ghost** [reply]
Thank you @aubreyd! I'll wait to proceed until you have a definitive answer, but from a cursory look that seems like the right information. That file also has some other variables that I was hoping to find too, which is great!
![image](https://github.com/user-attachments/assets/9e055c9c-24b4-46db-a381-41c3043f4f11)
